src/components/App/index.js

```diff
@@ -1,39 +1,63 @@
 import React, {Component} from 'react';
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+// through this react-redux bridge this component gets promoted to a container which is a "smart component"
+// that is a componnet that has direct connection to Redux
 import { today } from 'constants/apiQueries';
 
 import { fetchEpisodes } from 'redux/actions/episodesActions';
 
+import ActiveShowModal from 'components/App/ActiveShowModal';
 import EpisodesList from 'components/App/EpisodesList';
 import Header from 'components/App/Header';
+import Pagination from 'components/App/Pagination';
 import Spinner from 'components/App/Spinner';
 
 import styles from './App.module.scss';
 
 class App extends Component {
   componentDidMount() {
-    this.props.dispatch(fetchEpisodes());
+    this.props.fetchEpisodes();
   }
 
   render() {
-    const { error, loading, episodes } = this.props;
+    const { error, loading, episodes, activeShow } = this.props;
+    const { currentPage } = this.props.currentPage;
+    const episodesPerPage = 12; //arbitrary number, fits nicely into various screen widths
```

Comment:
If the number of episodes per page fits almost always why not hardcode it inside pagination? Or change dynamically according to screen size?